save-common/src/nativeMain/kotlin/org/cqfn/save/core/utils/PlatformUtils.kt

```diff
@@ -29,6 +31,21 @@ actual class AtomicInt actual constructor(value: Int) {
     actual fun addAndGet(delta: Int): Int = atomicInt.addAndGet(delta)
 }
 
+private fun String.escapePercent(): String {
+    val stringBuilder = StringBuilder("")
+    this.forEachIndexed { index, char ->
+        // last index in the string: aaa% -> aaa%
```

Comment:
For me it is not really clear explanation
```suggestion
        // We will escape the percent symbol in the message, except the cases, when the `%` is last symbol of the string or followed by another `%` symbol:
        // aaa% -> aaa%
        // %%aaa -> %%aaa
        // %aaa -> %%aaa
```
Or something like that